### ISSUE TITLE
createSession with error response `Too many requests`

### ISSUE THREAD

**Issue by kutear** [appwrite/appwrite#1085]
## 🐛 Bug Report

flutter sdk createSession with error
```
appwrite                        | [Error] Timestamp: 2021-04-11T17:42:22+00:00
appwrite                        | [Error] Method: POST
appwrite                        | [Error] URL: /v1/account/sessions
appwrite                        | [Error] Type: Utopia\Exception
appwrite                        | [Error] Message: Too many requests
appwrite                        | [Error] File: /usr/src/code/app/controllers/shared/api.php
appwrite                        | [Error] Line: 71
```

## Have you spent some time to check if this issue has been raised before?

YES

## To Reproduce

appwrite version 0.7.2 
appwrite flutter 0.4.0

## Expected behavior

login fail

## Actual Behavior

login success

## Your Environment

appwrite install on ubuntu 20.04

**Comment by eldadfux** [comment]
Hey @kutear, the `createSession` endpoint is limited to 10 request per hour for each unique email address. This is meant to protect your endpoint from malicious users trying to hack an account.

As mentioned here:
https://appwrite.io/docs/client/account?sdk=web#accountCreateSession
![image](https://user-images.githubusercontent.com/1297371/114316800-d4ae3980-9b0d-11eb-9a12-4d166b4fbca5.png)

You can disable the abuse protection with the `_APP_OPTIONS_ABUSE ` environment variable. Note that this option is **not** recommended for production usage. You can learn more about it here: https://appwrite.io/docs/environment-variables#general

**Comment by kutear** [comment]
![image](https://user-images.githubusercontent.com/8605389/114336764-3c9e6780-9b82-11eb-814a-0f0625e5bfe9.png)
@eldadfux 
But i never reach limit, and i never login success.

**Comment by lohanidamodar** [comment]
@kutear If you make 10 failed requests due to incorrect credentials, that will also be counted towards the limit. I believe before getting to many request errors, you must be getting some other invalid requests.

**Comment by kutear** [comment]
@lohanidamodar 
I created other user with different email, and get same error when i login.  it's because my email stutus is `unverified`?

**Comment by eldadfux** [comment]
Status `unverified` is not causing 429 errors. Any chance someone else on your server is making those bad requests and not you?

**Comment by kutear** [comment]
@eldadfux  no, only i use my server and do request, I checked my server logs, just had my request record.

**Comment by kutear** [comment]
![image](https://user-images.githubusercontent.com/8605389/114423528-d77e5c80-9be9-11eb-8777-3b484a729338.png)
So weird. I never requet from last one hour, but I still get this error, and http header show it's remain 0.
![image](https://user-images.githubusercontent.com/8605389/114424213-7acf7180-9bea-11eb-9e0e-26f124c08b1d.png)

**Comment by eldadfux** [comment]
@kutear are you sure you don't any script/page that is trying to constantly login? maybe you have a login call on app init?

**Comment by kutear** [comment]
> @kutear are you sure you don't any script/page that is trying to constantly login? maybe you have a login call on app init?

base on the logs of my server, without other's invoke, only had one my record.
you can see my shotscreen, from `12:51` to `15:38`, only had a few request. and i still get error at 15:38.

**Comment by eldadfux** [comment]
@kutear don't really have any more ideas. I can't think of a reason why the server will generate API calls just like that on an isolated environment. Can you try and check what IP is generating the calls?

**Comment by TorstenDittmann** [comment]
This got fixed now with #1434 🙌🏻
